koku/masu/test/util/aws/test_common.py

```diff
@@ -683,10 +683,10 @@ def test_match_openshift_resources_and_labels(self):
 
         # resource id matching
         result = matched_df[matched_df["lineitem_resourceid"] == "id1"]["resource_id_matched"] == True  # noqa: E712
-        self.assertTrue(result.bool())
+        self.assertTrue(result.any())
```

Comment:
Should these also be `.any(bool_only=True)`?

Reply:
This is a `Series` object and `only_bool` isn't implemented on Series objects (it is for DataFrames).

It actually looks like this should be `all()`. `result` is filtered to only values with `True`. The `.bool()` method used to ambiguously mean "only one True item". Maybe it is most accurate to assert the long is one and `.all()`.

Reply:
Took me a minute, but I now see the `== True` is a [boolean reduction](https://pandas.pydata.org/pandas-docs/stable/user_guide/basics.html#boolean-reductions). Calling `.any().any()` will reduce to a final boolean value.